### PR TITLE
Read pages with context

### PR DESCRIPTION
**Pull request by jlee-enquizit**
Here we are reading CRUD operations and storing the params in state, establishing a simpler pattern for all PageBuilder sections.
This should establish a pattern where we don't need to call the `usePage` hook with tables as well.

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePages.tsx [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
-                                <ManagePagesTableWrapper />
-                            </PageProvider>
+                            {pages && !isLoading && (
+                                <ManagePagesTable
+                                    summaries={pages}
+                                    currentPage={currentPage}
+                                    pageSize={pageSize}
+                                    totalElements={totalElements}
+                                />
+                            )}
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePages.tsx:42]: do we want to hide the table while loading or is there any support for a loading indicator?

File: apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.spec.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { PageProvider } from 'page';
+// import { PageProvider } from 'page';
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.spec.tsx:1]: commented out import

File: apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.tsx [hunk 3]
```diff
@@ -27,12 +27,15 @@ const tableColumns = [
 ];
 
 type Props = {
-    sortChange: (sort?: string) => void;
+    // sortChange: (sort?: string) => void;
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.tsx:30]: commented method

File: apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePages.tsx (hunk 1 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePages.tsx:42]: Added the Spinner

**Review by hclarkEnq — APPROVED** [review]

**Pull request merged**